AutomationPractice/features/Pages/CreateAccountPage.py

```diff
@@ -5,13 +5,13 @@
 from selenium.webdriver.support import expected_conditions as EC
 from ultilities.Random import Random
 import Locator
+from features.browsers.browser import Browser
 
 
-class CreateAccount():
+class CreateAccount(Browser):
     rd = Random()
     def __init__(self, driver):
         self.driver = driver
-
```

Comment:
.Please follow coding conventions PEP8 for python.

Reply:
fixed as above comment